lib/oli_web/live/sections/enrollments/enrollments_view.ex

```diff
@@ -121,9 +133,36 @@ defmodule OliWeb.Sections.EnrollmentsViewLive do
 
   def render(assigns) do
     ~F"""
-    <div class="container mx-auto">
-
-      <div class="d-flex justify-content-between">
+    <div id="enrollments_view" class="container mx-auto">
+      {#if @is_admin}
```

Comment:
In addition to only showing this when the user is an Admin, we only want to show this feature for Direct Delivery course sections.  LTI course sections cannot support this email based enrollment.

Reply:
Done!